ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -28,11 +28,12 @@ install_ipa_client: false
 
 # Enable this if you want to create a user on the bastion
 # Mutually exclusive with {{ install_ipa_client }}
-install_student_user: true
+install_student_user: false
```

Comment:
Please revert this change and do not include another config in this PR. If this needs to change, please open an issue or submit a separate PR so the config owner can address.

Reply:
ocp4-cluster ain't mine/not using those in my config, these files have been pushed by mistake, sorry for that